doc/conf.py

```diff
@@ -291,3 +292,14 @@ def setup(app):
                                          u'https://github.com/scikit-learn/'
                                          'scikit-learn/blob/{revision}/'
                                          '{package}/{path}#L{lineno}')
+
+
+# Removes 'any' reference target not found from logging
+def filter_any_references(record):
+    msg = record.getMessage()
+    return ("'any' reference target not found:" not in msg and
+            "more than one target found for 'any' cross-reference" not in msg)
```

Comment:
Can you give examines of this second error from the logs please?

Reply:
```
scikit-learn/sklearn/compose/_column_transformer.py:docstring of 
sklearn.compose.make_column_transformer:15: 
WARNING: more than one target found for 'any' cross-reference 'fit': 
could be :std:term: or :py:meth: or :py:meth: ...
```

Looking at the html, this `fit` reference does not actually reference anything. I suspect we want this to reference to the glossary. 

This PR was updated to remove the second error filter, given it is a valid error.